packages/components/src/styles/index.css

```diff
@@ -1,5 +1,5 @@
 /* purgecss start ignore */
-@import url("https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap");
+@import "./fonts.css";
```

Comment:
what's the benefit/needs for this change?

Reply:
Instead of playing with font-weight (300, 400, 500, 700) we can define a font-face name as Roboto-Light, Roboto-Regular etc..

The content of fonts.css is exactly the same output generated by opening `url("https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap")`.